doc.html

```diff
@@ -16,1281 +16,982 @@
+                            Additional tools can be registered:
+                        </p>
+                        <p class="c_snip">
+                            pbench-register-tool --name blktrace
+                        </p>
+                        <p class="info_text" style="text-align:justify">
+                            or unregistered (e.g. some people prefer to run without perf):
+                        </p>
+                        <p class="c_snip">
+                            pbench-unregister-tool --name perf
+                        </p>
+                        <p class="info_text" style="text-align:justify">
+                            Note that perf is run in a "low overhead" mode with options
+                            "record -a –freq=100", but if you want to run it differently, you can
+                            always unregister it and register it again with different options:
+                        </p>
+                        <p class="c_snip">
+                            pbench-unregister-tool --name=perf
+                            <br>pbench-register-tool --name=perf -- --record-opts="record -a
+                            --freq=200"
+                        </p>
+                        <p class="info_text" style="text-align:justify">
+                            Tools can be also be registered, started and stopped on remote hosts
+                            (see the --remote option described in What does --remote do? in <a
+                                href="learn.html">FAQ section</a>
+                        </p>
+                    </div>
+                    <div id="availablescript" class="section_cardbody">
+                        <p class="cardHeader">Available benchmark scripts</p>
+                        <p class="info_text" style="text-align:justify">Pbench provides a set of
+                            pre-packaged scripts to run some common benchmarks using the collection
+                            tools and other facilities
+                            that pbench provides. These are found in the bench-scripts directory of
+                            the Pbench installation (/opt/pbench-agent/bench-scripts by default).
+                            The current set includes:
+                        </p>
+                        <div class="info_text" style="text-align:justify">
+                            <li> pbench-dbench </li>
+                            <li> pbench fio </li>
+                            <li> pbench-linpack</li>
+                            <li> pbench-migrate </li>
+                            <li> pbench-tpcc</li>
+                            <li> pbench-uperf </li>
+                            <li>pbench-user-benchmark (see <a href="#runningtools">Running Pbench
+                                    collection tools with an arbitrary benchmark </a>below for more
+                                on this)</li>
```

Comment:
Can we wrap this against a `<ul>` tag and reformat the `<li>` on a newline

Reply:
fixed